grafe-cli/src/generate/generate.ts

```diff
@@ -89,22 +182,20 @@ export function generateCLI() {
  * @param description The description of the middleware
  * @returns if everything correct, creates new file
  */
-export async function generateMiddleWare(name: string, short: string, description: string) {
+export async function generateMiddleWare(name: string, short: string, description: string): Promise<void> {
     const rootDir = await pkgDir(process.cwd());
 
     let raw;
     try {
-        raw = fs.readFileSync(rootDir + "/grafe.json");
+        raw = fs.readFileSync(path.join(rootDir, 'grafe.json'));
     } catch (err) {
-        console.log("The grafe command must be used within a grafe project.");
-        return;
+        return console.log("The grafe command must be used within a grafe project.");
     }
 
     let data = JSON.parse(raw.toString());
 
     if (data.middlewares.some((item: any) => item.name === name) || data.middlewares.some((item: any) => item.value === short)) {
-        console.log("Either the name or the shortcut of this middleware is already in use");
-        return;
+        return console.log("Either the name or the shortcut of this middleware is already in use");
```

Comment:
Mach da am besten zwei ifs draus damit der dude der das benutzt weiß was der fehler is